app/templates/macros/toolkit_forms.html

```diff
@@ -1,9 +1,9 @@
-{% macro text(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
The problem is that because the macros are called dynamically they have to all accept the same arguments, so without using `kwargs` in order to add a new argument to be used in one macro we have to add it to the argument list of all macros.  So this seems the exact kind of thing that `kwargs` is designed for?